diff --git a/Project.toml b/Project.toml
index 10ab354..1f0e92d 100644
--- a/Project.toml
+++ b/Project.toml
@@ -28,26 +28,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 TestSetExtensions = "98d24dd4-01ad-11ea-1b02-c9a08f80db04"
 
 [compat]
-BaryRational = "1"
-DifferentialEquations = "7"
-ForwardDiff = "0.10"
-Groebner = "0.6, 0.7, 0.8"
-HomotopyContinuation = "2"
-LinearSolve = "2"
-ModelingToolkit = "9"
-Nemo = "0.40 - 0.48"
-OrderedCollections = "1"
-PrecompileTools = "1"
-ProgressMeter = "1"
-SIAN = "1.6"
-StructuralIdentifiability = "0.5"
-Suppressor = "0.2"
-Symbolics = "4, 5, 6"
 TaylorDiff = "0.2"
-TaylorSeries = "0.12 - 0.30"
-Test = "1.10"
-TestSetExtensions = "2"
-julia = "1"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
