diff --git a/Project.toml b/Project.toml
index 929e97b..3b83bab 100644
--- a/Project.toml
+++ b/Project.toml
@@ -12,12 +12,6 @@ MPFI = "e50a78b8-6e2f-482e-b7c3-776aa2235d1a"
 [sources]
 MPFI = {url = "https://gitlab.inria.fr/ckatsama/mpfi.jl.git"}
 
-[compat]
-AbstractAlgebra = "0.41, 0.42, 0.43, 0.44"
-Groebner = "0.9.5"
-MPFI = "0.0.2"
-julia = "1.6.7"
-
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
